consensus/clique/clique.go

```diff
@@ -191,16 +191,15 @@ type Clique struct {
 // signers set to the ones provided by the user.
 func New(config *params.CliqueConfig, db ethdb.Database) *Clique {
 	// Set any missing consensus parameters to their defaults
-	conf := *config
```

Comment:
Any reason they added this `conf := *config` line before?

Reply:
Don't know any suitable reason. `config` is not changed in any places inside `clique` struct